engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/transform/query/NadelQueryTransformer.kt

```diff
@@ -19,7 +19,7 @@ class NadelQueryTransformer internal constructor(
         suspend fun transform(fields: List<ExecutableNormalizedField>): List<ExecutableNormalizedField>
     }
 
-    internal data class TransformContext(
+    data class TransformContext(
```

Comment:
Any reason you removed `internal`? We want `internal` here because this is NOT public API

Reply:
thanks, have no idea why I did that